### PR TITLE
Fix links, seed, and flatten

### PR DESCRIPTION
**Pull request by brahmaneya**
## Description of proposed changes
Several small fixes in tutorials

## Test plan
Existing tests

## Checklist

Need help on these? Just ask!

* [x] I have read the **CONTRIBUTING** document.
* [x] I have verified that my changes are covered by continuous integration.
* [x] All new and existing tests passed.

### CODE REVIEW

File: multitask/multitask_tutorial.py [hunk 1]
```diff
@@ -26,10 +26,10 @@
 # %% {"tags": ["md-exclude"]}
 # %matplotlib inline
 
-from snorkel.utils import set_seed
+import torch
 
 SEED = 123
-set_seed(SEED)
+torch.manual_seed(SEED)
```
Comment (vincentschen) [inline multitask/multitask_tutorial.py]: is there an associated numpy seed to set?
